staging/src/k8s.io/apimachinery/pkg/util/yaml/decoder.go

```diff
@@ -215,16 +213,15 @@ type YAMLOrJSONDecoder struct {
 	bufferSize int
 
 	decoder decoder
-	rawData []byte
 }
 
 type JSONSyntaxError struct {
-	Line int
```

Comment:
Can you leave Line here with a note that it isn't populated currently?

Reply:
Done (though I think it will only create confusion for one who would like to use it).

Reply:
Yeah I can see going either way, I was mostly thinking in terms of not breaking anyone using this public type at compile time. I can see people thinking that'd actually be a good thing. In this case I lean towards keeping it, but I don't feel super strongly.

Reply:
I removed it